roles/ceph-mon/defaults/main.yml

```diff
@@ -42,16 +41,16 @@ calamari: false
 openstack_config: false
 openstack_glance_pool:
   name: images
-  pg_num: "{{ pool_default_pg_num }}"
+  pg_num: "{{ osd_pool_default_pg_num }}"
 openstack_cinder_pool:
   name: volumes
-  pg_num: "{{ pool_default_pg_num }}"
+  pg_num: "{{ osd_pool_default_pg_num }}"
 openstack_nova_pool:
   name: vms
-  pg_num: "{{ pool_default_pg_num }}"
+  pg_num: "{{ osd_pool_default_pg_num }}"
 openstack_cinder_backup_pool:
   name: backups
-  pg_num: "{{ pool_default_pg_num }}"
+  pg_num: "{{ osd_pool_default_pg_num }}"
```

Comment:
Won't this fail in the same way because where is ``osd_pool_default_pg_num`` defaulted?